react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+export function ToggleSwitch() {
+  const [onOrOff, setOnOrOff] = useState('toggle-switch is-on');
+  const [labelTxt, setLabelTxt] = useState('On');
```

Comment:
Since you can compute the label from your `onOrOff` state, you should do that instead of having a separate `labelTxt` state.